prow/plank/reconciler.go

```diff
@@ -814,9 +814,6 @@ func (r *reconciler) canExecuteConcurrentlyPerQueue(ctx context.Context, pj *pro
 	if !queueDefined {
 		return false, fmt.Errorf("failed to match queue name '%s' with Plank configuration", queueName)
 	}
-	if queueConcurrency == 0 {
-		return true, nil
-	}
```

Comment:
Can't we return `false, nil` right away and save some API calls that will result in the same thing?

Reply:
Done.